tests/testthat/indention_operators/eq_formals_complex_indention-out.R

```diff
@@ -0,0 +1,33 @@
+function(a =
+           33,
+         b
+) {}
+
+function(a =
+           33,
+         b) {}
+
+function(a,
+         b,
+         c
+) {}
+
+function(a,
+         b,
+         c) {}
```

Comment:
Add line break if formals consume multiple lines?

Reply:
Line break for the closing brace? I think under [long lines](http://style.tidyverse.org/functions.html#long-lines-1), there is no line break for that case (just an example, not covered in the floating text).
Should we open an issue to make the tidyverse style guide explicit in this regard?

Reply:
Never mind, it's not that important.

Reply:
Ok, let's leave that one for now